### PR TITLE
12/lander

### PR DESCRIPTION
**Pull request by BlackHawk0**
# Description

The page provide an intro the elewa impact. The page is both web and mobile resposinve thus providing a good user experience.

## Type of change
- [X] New feature (non-breaking change which adds functionality)

# Screenshot (optional)

image

image

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: libs/features/components/banners/src/lib/elewa-group-image-and-text-banner/elewa-group-image-and-text-banner.component.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<div class="container" [ngStyle]="{'background-color': bgcolor}">
```
Comment (LemmyMwaura) [inline libs/features/components/banners/src/lib/elewa-group-image-and-text-banner/elewa-group-image-and-text-banner.component.html:1]: This component has already been implemented. Sync the latest changes and work with what is already there.

File: libs/pages/elewa/home/src/lib/components/impact/impact.component.scss [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    margin-right: 30px;
+  }
+
+}
+
+
+$breakpoint-tablet: 768px;
+$breakpoint-low-res-desktops: 1360px;
+
+@media (max-width: $breakpoint-low-res-desktops){
```
Comment (LemmyMwaura) [inline libs/pages/elewa/home/src/lib/components/impact/impact.component.scss:41]: Would be perfect if breakpoints are saved in a separate scss file but this works for now. Good 👍

File: libs/pages/elewa/home/src/lib/features-elewa-home.module.ts [hunk 3]
```diff
@@ -15,25 +15,34 @@ import { PrevDirective } from './directives/prev.directive';
 import { NextDirective } from './directives/next.directive';
 
 import { HomeRoutingModule } from './home.routing';
-
-
+import { ImpactComponent } from './components/impact/impact.component';
+import { ButtonsModule } from '@elewa-group/features/components/buttons';
```
Comment (LemmyMwaura) [inline libs/pages/elewa/home/src/lib/features-elewa-home.module.ts:19]: Order of imports

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]

**Comment by LemmyMwaura** [comment]
Learn More button should also be full-width on the mobile version but good job so far.

File: libs/pages/elewa/home/src/lib/features-elewa-home.module.ts (hunk 3 above)
Comment (BlackHawk0) [reply libs/pages/elewa/home/src/lib/features-elewa-home.module.ts:19]: resolved by grouping components together and modules together

File: libs/pages/elewa/home/src/lib/components/impact/impact.component.scss (hunk 2 above)
Comment (BlackHawk0) [reply libs/pages/elewa/home/src/lib/components/impact/impact.component.scss:41]: sure. Thanks

**Comment by BlackHawk0** [comment]
> Learn More button should also be full-width on the mobile version but good job so far.

corrected

File: libs/features/components/banners/src/lib/elewa-group-image-and-text-banner/elewa-group-image-and-text-banner.component.html (hunk 1 above)
Comment (BlackHawk0) [reply libs/features/components/banners/src/lib/elewa-group-image-and-text-banner/elewa-group-image-and-text-banner.component.html:1]: Yes the component has already been implemented maybe i just added a space by mistake but i have not modified it

**Comment by IanOdhiambo9** [comment]
Update screenshot

**Comment by IanOdhiambo9** [comment]
Also confirm that when a user hovers over one button it does not trigger the animation for both buttons. it's a common issue I've being seeing on other PRs

**Review by IanOdhiambo9 — COMMENTED** [review]
This PR cannot be merged because of the package-lock.json file you've pushed

**Comment by IanOdhiambo9** [comment]
I'm closing this. 
Implement changes a do a new PR, just copy description to the new PR with updated screenshots.

Without the package-lock.json file.

**Pull request closed without merging**